### PR TITLE
[FUSEQE-1425] Stabilize REST + cleanup

### PR DESCRIPTION
**Pull request by avano**
//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/utils/FtpUtils.java [hunk 1]
```diff
@@ -1,108 +1,59 @@
...
-        } else {
-            log.info("UPLOAD FAILED!");
-            return false;
+    public void uploadTestFile(String testFileName, String text, String remoteDirectory) {
+        checkConnection();
+        log.info("Uploading file " + testFileName + " with content " + text + " to directory " + remoteDirectory + ". This may take some time");
+        try (InputStream is = IOUtils.toInputStream(text, "UTF-8")) {
+            ftpClient.storeFile("/" + remoteDirectory + "/" + testFileName, is);
+        } catch (IOException ex) {
+            ex.printStackTrace();
```
Comment (mcada) [inline utilities/src/main/java/io/syndesis/qe/utils/FtpUtils.java]: Do we want to add this stacktrace into logger?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**